tests/p2p/config.yaml

```diff
@@ -10,7 +10,7 @@ deployment_ready_time_out: 120 #default 120
 config_path: '../config.toml'
 
 bootstrap:
-  image: 'spacemeshos/go-spacemesh:develop'
+  image: 'spacemeshos/go-spacemesh-dev:go_auto'
```

Comment:
please change back to develop

Reply:
207fd37bd289dcc691ce4b4207a102bc0f009af7